cs6232-g4/Controller/TransactionController.cs

```diff
@@ -67,5 +67,22 @@ public List<double> CreateReturnTransaction(List<RentalLineItem> rentalLineItems
         {
             return this._transactionDAL.CreateReturnTransaction(rentalLineItems);
```

Comment:
It isn't returning the history
![Screenshot 2024-04-17 at 8 20 14 PM](https://github.com/lk00044/MLJFurnitureStore/assets/112569246/74e1b58e-a7b2-476d-bd26-4c8608f98ae7)

Reply:
I did some debugging. I believe you are just triggering the form in SearchForMemberUserControl.cs without any actions triggering to retrieve data.